recipes/edyn/all/conanfile.py

```diff
@@ -24,80 +25,71 @@ class EdynConan(ConanFile):
         "fPIC": True,
         "floating_type": "float",
     }
-    generators = "cmake", "cmake_find_package_multi"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
-
-    def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
-
-    def config_options(self):
-        if self.settings.os == "Windows":
-            del self.options.fPIC
-
-    def configure(self):
-        if self.options.shared:
-            del self.options.fPIC
 
     def requirements(self):
-        self.requires("entt/3.9.0")
+        if scm.Version(self.version) < "1.2.0":
+            self.requires("entt/3.9.0")
+        else:
+            self.requires("entt/3.10.1")
 
     @property
     def _compiler_required(self):
         return {
             "gcc": "9.3", # GCC 9.3 started supporting attributes in constructor arguments
+            "clang": "8",
+            "apple-clang": "10",
+            "Visual Studio": "16",
+            "msvc": "192"
```

Comment:
there's a check min vs helper that now exists ( i will circle back after my distraction to add a link )

Reply:
https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#check-min-vs

Reply:
Would this need to be handled in an if branch for VS/msvc? Or does it silently pass for non-mvsc compilers?